ruby-challenges.rb

```diff
@@ -10,6 +10,25 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+# PROCESS:
+# use def for a new method called odd_or_even
+# use modulo num % 2 == 0 to determine if odd or even
+# use string interpolation #{} on num for our return to grab the value and put it in our string "#{num} is odd/even"
+# print new method with each num
+
+def odd_or_even(num)
+    if num % 2 == 0 then
```

Comment:
Awesome work!  Ruby also has a built-in method called .even? that will do some of this work for you.